.github/workflows/test_package.yml

```diff
@@ -9,16 +9,18 @@ jobs:
     name: unit-tests
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        python-version: [3.6]  #, 3.7, 3.8]
-        pytorch-version: ["1.8.0", "nightly"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        pytorch-version: ["stable", "nighly"]
```

Comment:
Let's keep a fixed version of PyTorch (instead of stable) besides nightly

Reply:
1.8.0, because it is the oldest supported one?
2.5.1 because it is the latest stable release?

Reply:
2.5.1 and nighly will be perfect.